### ISSUE TITLE
No field "session_id" in "/v2/appointment/sessions/public/calendarByCenter" api

### ISSUE THREAD

**Issue by sablearjun** [cowinapi/developer.cowin#517]
session_id field is not returned in calendarByCenter api. Please fix it.

**Comment by gampu** [comment]
Yes. Please make `session_id` field available in `calendarByCenter` API. It is cumbersome to obtain the same from `findByPin` API. 
@sablearjun Are there any better/easy alternatives for obtaining the same?
